### ISSUE TITLE
Show game download size on the game info page.

### ISSUE THREAD

**Issue by sergiotapia** [PCMRating/pcmrating#60]
This is something I was going to make a standalone website for but it will fit in much nicer in PCMRating. Let's show how large a game's download size is (obviously on Steam).

So say something like The Witcher 3 - let's show a `39GB` somewhere on the game's show page.

**Comment by tjhorner** [comment]
How would we get this, user input? There aren't any Steam APIs that give us the size of a game. Even SteamDB doesn't have this.

**Comment by Lockyy** [comment]
Yeah, gathering the data does seem to be the issue here. If you can find an API that will give this idea we'll get it in.

**Comment by sergiotapia** [comment]
We can scrape the info from this page: https://steamdb.info/app/105600/depots/

Some games like Dota 2 have many depots so we need to write some code to order them and grab the largest depo.

https://steamdb.info/app/570/depots/

This is kinda simple to do and we only need to parse this info when someone manually adds a game.

**Comment by tjhorner** [comment]
No. We can't scrape SteamDB: https://steamdb.info/faq/#can-i-use-auto-refreshing-plugins-or-automatically-scrape-crawl-steamdb
